third_party/amd/lib/TritonAMDGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -1227,6 +1241,7 @@ struct AsyncTDMCopyLocalToGlobalOpConversion
     auto tensorDescTy = op.getDesc().getType();
     auto smemTy = op.getSrc().getType();
     Type elementType = getTypeConverter()->convertType(smemTy.getElementType());
+    int numCTAs = getNumCTAs(smemTy.getEncoding());
```

Comment:
While this is correct, let's use the one that gets it from the context. We should remove this `getNumCTAs` utility and just have one source of truth.